UI/uiModelComparison.py

```diff
@@ -0,0 +1,62 @@
+'''
+file implementing the UI of model performance comparison of an image
+'''
+
+# --- /
+# -- / external imports
+import streamlit as st
+import PIL
+
+
+def compareModels(confidenceThreshold:float):
```

Comment:
looks good, however It might be useful to select sample images too, would it not?
So adding the option to either choose a pre-existing image or some uploaded by the user